ui/StatusQ/src/StatusQ/Controls/StatusNavBarTabButton.qml

```diff
@@ -1,13 +1,16 @@
 import QtQuick 2.13
+import StatusQ.Core 0.1
 import StatusQ.Components 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Popups 0.1
+import StatusQ.Core.Theme 0.1
     
 StatusIconTabButton {
     id: statusNavBarTabButton 
     property alias badge: statusBadge
     property alias tooltip: statusTooltip
     property Component popupMenu
+    property bool isMemberBanned: false
```

Comment:
I'm thinking that `StatusNavBarTabButton` is kind of generic component and I don't think we should add "logical / specific" behaviour in here. 

WDYT if instead of adding this `isMemberBanned` property in here you add something more generic?
- Rename the property with a generic name (`topIconVisible` or `stateIconVisible`).
- Expose the `asset` property for this icon so the consumer can decide different icons.
- Expose `background` and `border` colors to be more flexible.

Reply:
changed